### PR TITLE
Implement replace

### PR DESCRIPTION
**Pull request by smallcreep**

### CODE REVIEW

File: lib/fluent/plugin/filter_record_modifier.rb [hunk 1]
```diff
@@ -66,6 +71,20 @@ def configure(conf)
         end
       end
 
+      @replace_keys = Array.new
+      if @replace
+        conf.elements.select { |element| element.name == 'replace' }.each do |element|
```
Comment (repeatedly) [inline lib/fluent/plugin/filter_record_modifier.rb]: You can use `config_section` for it.
See rewrite-tag-filter code: https://github.com/fluent/fluent-plugin-rewrite-tag-filter/blob/9565ddf978faf0d419ffa3a796cbf8cbd292a9af/lib/fluent/plugin/out_rewrite_tag_filter.rb#L15

File: lib/fluent/plugin/filter_record_modifier.rb [hunk 2]
```diff
@@ -66,6 +71,20 @@ def configure(conf)
         end
       end
 
+      @replace_keys = Array.new
```
Comment (repeatedly) [inline lib/fluent/plugin/filter_record_modifier.rb]: `@replace_keys = nil` and set `Array.new` inside `if @replace`

File: lib/fluent/plugin/filter_record_modifier.rb [hunk 3]
```diff
@@ -66,6 +71,20 @@ def configure(conf)
...
+          expr = if element['expression'][0] == "/" && element['expression'][-1] == "/"
+                   element['expression'][1..-2]
+                 else
+                   element['expression']
+                 end
+          @replace_keys.push("key" => element['key'],
+                             "expression" => Regexp.new(expr),
+                             "replace" => element['replace'])
+        end
+      end
```
Comment (repeatedly) [inline lib/fluent/plugin/filter_record_modifier.rb]: if `@replace_keys` is empty, should raise `Fluent::ConfigError`.

File: lib/fluent/plugin/filter_record_modifier.rb [hunk 4]
```diff
@@ -97,6 +116,12 @@ def filter(tag, time, record)
         record = modified
       end
 
+      if @replace && @replace_keys
```
Comment (repeatedly) [inline lib/fluent/plugin/filter_record_modifier.rb]: Check only `@replace_keys` is enough.

File: lib/fluent/plugin/filter_record_modifier.rb (hunk 1 above)
Comment (smallcreep) [reply lib/fluent/plugin/filter_record_modifier.rb]: Thx. I rewrote code with using `config_section`.

**Comment by smallcreep** [comment]
@repeatedly Please check this PR.

**Comment by repeatedly** [comment]
Sorry for the late response. Just merged.

**Pull request merged**